### PR TITLE
Ros2 add save map service

### PR DESCRIPTION
**Pull request by FrankBu0616**
Hi, I modified the ros2 branch a bit to add the save_map service. I also modified the README on how to use it. I tested it with my ouster lidar as well. The service callback function is copied from the ros1 version verbatim. Thank you for the wonderful package.

### CODE REVIEW

**Comment by grischi** [comment]
Hi! Thank you for your contribution! Would you mind fixing the whitespace errors?

**Comment by FrankBu0616** [comment]
Hi! Sorry about that. Is it better now? I tried to remove useless empty lines.

File: srv/SaveMap.srv [hunk 1]
```diff
@@ -0,0 +1,4 @@
+float32 resolution
+string destination
+---
+bool success
```
Comment (grischi) [inline srv/SaveMap.srv:4]: Here's a newline missing at the EOF.

Comment (FrankBu0616) [reply srv/SaveMap.srv]: Fixed it! Totally missed that.

**Review by grischi — APPROVED** [review]

**Pull request merged**